flink-runtime/src/main/java/org/apache/flink/runtime/checkpoint/CheckpointStatsTracker.java

```diff
@@ -112,8 +112,9 @@ public class CheckpointStatsTracker {
      *     progress ones.
      * @param metricGroup Metric group for exposed metrics
      */
-    public CheckpointStatsTracker(int numRememberedCheckpoints, MetricGroup metricGroup) {
-        this(numRememberedCheckpoints, metricGroup, new JobID(), Integer.MAX_VALUE);
+    public CheckpointStatsTracker(
```

Comment:
nit: How about also adding "@param" for jobID since it has others?
Or just introducing a new method for Tests so that they could not be modified.

Reply:
Makes sense, added